kolibri/plugins/coach/assets/src/views/class-list-page.vue

```diff
@@ -18,8 +18,7 @@
         <caption class="visuallyhidden">{{ $tr('tableCaption') }}</caption>
         <thead slot="thead">
           <tr>
-            <th class="core-table-icon-col"></th>
```

Comment:
Do we want the name column to also contain the icon?

Reply:
I think that could work, too. but yeah the goal here was to get all the headers left-aligned with the page